profiling-ffi/src/exporter.rs

```diff
@@ -180,6 +181,7 @@ pub unsafe extern "C" fn profile_exporter_build(
     end: Timespec,
     files: Slice<File>,
     additional_tags: Option<&ddcommon_ffi::Vec<Tag>>,
+    endpoints_stats: Option<&profiled_endpoints::ProfiledEndpointsStats>,
```

Comment:
An earlier version used a `Box` here, and this has now become a `&`. I'm actually not sure what the difference is -- @morrisonlevi is this fine in this situation? (Asking just in case)

Reply:
We already do the same, for example profile or exporter object. 
- We create the pointer (with `Box`)
- Pass it to the library
- The library do calls passing the profiler/exporter, we use `&` instead
- In the `free` or `drop`, we box it again to release the memory

Reply:
Thanks for checking!